go.mod

```diff
@@ -45,11 +43,10 @@ require (
 	gopkg.in/fsnotify.v1 v1.4.7
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0-20170531160350-a96e63847dc3
-	k8s.io/api v0.0.0-20191004102349-159aefb8556b // kubernetes-1.14.10
-	k8s.io/apiextensions-apiserver v0.0.0-20191212015246-8fe0c124fb40 // kubernetes-1.14.10
-	k8s.io/apimachinery v0.0.0-20191004074956-c5d2f014d689 // kubernetes-1.14.10
-	k8s.io/client-go v11.0.1-0.20191029005444-8e4128053008+incompatible // kubernetes-1.14.10
+	k8s.io/api v0.15.11 // kubernetes-1.14.10
```

Comment:
I'm not sure where these comments came from (did we add them by hand) but they are now out of date.

Reply:
go.mod is generated by the go tooling. I've bo idea where they came from.

I'll edit them by hand.

Reply:
Done!